region/norfair/east/Spiky Platforms Tunnel.json

```diff
@@ -138,6 +138,52 @@
       "flashSuitChecked": true
     },
     {
+      "link": [1, 1],
+      "name": "X-Mode Shinecharge, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_heatProof",
+        "h_RModeCanRefillReserves",
+        {"enemyKill": {"enemies": [["Tripper"]]}},
+        {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+        {"or": [
+          {"partialRefill": {"type": "ReserveEnergy", "limit": 5}},
```

Comment:
HeatTriggered interrupt is dicey on 5 energy.

Reply:
Since heatproof is already required, the pause abuse method should cover that. I should make that the primary path.